### PR TITLE
[Porkbun] fix: Domain Pricing would crash if user reloaded too many times via Action

### PR DESCRIPTION
**Pull request by xmok**
## Description

This is an issue from Extension Issues page.

porkbun-crashing

[Fix Domain Pricing Crashing]

- `Domain Pricing` command would crash if user tried to reload too many times. The "Reload Domain Pricing" action is now hidden if already fetching.

## Screencast

N/A

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/porkbun/src/utils/types.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ export type DomainPricing = {
           renewal?: Coupon;
           transfer?: Coupon;
         };
+    specialType?: string;
```
Comment (jfkisafk) [inline extensions/porkbun/src/utils/types.ts:20]: I don't see any usage for this atm. Is this necessary?

```suggestion
```

File: extensions/porkbun/src/domain-pricing.tsx [hunk 2]
```diff
@@ -97,10 +97,11 @@ export default function DomainPricing() {
       navigationTitle={navigationTitle}
       actions={
         <ActionPanel>
-          <Action icon={Icon.Redo} title="Reload Domain Pricing" onAction={callApi} />
+          {!isLoading && <Action icon={Icon.Redo} title="Reload Domain Pricing" onAction={callApi} />}
           <Action.OpenInBrowser icon={Icon.Globe} title="Go to API Reference" url={`${API_DOCS_URL}Domain%20Pricing`} />
         </ActionPanel>
       }
+      searchBarPlaceholder="Search domain pricing"
```
Comment (jfkisafk) [inline extensions/porkbun/src/domain-pricing.tsx:104]: To be more specific should we use this:
```suggestion
      searchBarPlaceholder="Search domain by name"
```

**Review by jfkisafk — APPROVED** [review]
LGTM. Just 2 minor questions.

File: extensions/porkbun/src/utils/types.ts (hunk 1 above)
Comment (xmok) [reply extensions/porkbun/src/utils/types.ts:20]: This is a new addition to the API. I planned to turn this into a "good first issue" via GitHub issues so someone who wants to try a new PR could add a Metadata Label item for it. But I can remove it.

**Review by jfkisafk — APPROVED** [review]

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**